src/xenia/base/memory.h

```diff
@@ -441,6 +441,10 @@ inline void store_and_swap<std::u16string>(void* mem,
   return store_and_swap<std::u16string_view>(mem, value);
 }
 
+inline constexpr uint32_t multi_char(char a, char b, char c, char d) {
```

Comment:
This name seems kinda ambiguous to me — specifically, this generates a four-character code, aka FourCC.